test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -6652,6 +6652,47 @@ def forward(self, input):
         input = torch.randn(2, 5, 7, dtype=torch.float64)
         self.run_test(Celu(), (input,))
 
+    def test_lower_tuple(self):
```

Comment:
(1) Is this PR handling the case with multiple tuple inputs? Can we have some unit tests about that to make sure it works as expected?
(2) Also, seems the code handles the case: one tuple has more than 2 elements: i.e., (t1, t2, t3), can we have a unit test to cover that?